src/client/App.js

```diff
@@ -5,28 +5,39 @@ import Meals from "./components/Meals";
 import Reservations from "./components/Reservations";
 import Reviews from "./components/Reviews";
 import About from "./components/About";
+import AddMeal from "./components/AddMeal";
+import AddReview from "./components/AddReview";
+import MealById from "./components/MealById";
+import ReviewById from "./components/ReviewById";
+import AutoSearch from "./components/AutoSearch";
+import AllMeals from "./components/All-meals";
 import { ContextProvider } from "./components/DataContext";
 import "./App.css";
 
 function App() {
   return (
     <Router className="app">
       <Switch>
-        <Route exact path="/home" component={HomePage} />
+        <Route exact path="/" component={HomePage} />
```

Comment:
One thing that stands out to me is how this is the only component here that is called Page, I would expect to see AboutPage and MealsPage etc. since they're all pages.